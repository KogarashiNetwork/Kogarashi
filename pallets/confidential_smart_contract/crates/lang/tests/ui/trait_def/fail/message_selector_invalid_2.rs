use ink_lang as ink;

#[ink::trait_definition]
pub trait TraitDefinition {
    #[ink(message, selector)]
    fn message(&self);
}

fn main() {}
