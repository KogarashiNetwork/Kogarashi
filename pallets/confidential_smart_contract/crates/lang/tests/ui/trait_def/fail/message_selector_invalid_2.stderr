error: encountered #[ink(selector)] that is missing its u32 parameter. Did you mean #[ink(selector = value: u32)] ?
 --> tests/ui/trait_def/fail/message_selector_invalid_2.rs:5:20
  |
5 |     #[ink(message, selector)]
  |                    ^^^^^^^^
